.github/workflows/CI.yml

```diff
@@ -26,12 +26,14 @@ jobs:
     env:
       SQLITE_VERSION : "3420000"
       SQLITE_YEAR: "2023"
+      POSTGIS_VERSION : "3.4.2"
       HTTP_PROXY: ""
       HTTPS_PROXY: ""
     strategy:
       fail-fast: false
       matrix:
         pg: ${{ fromJSON(needs.detect-pgversion.outputs.pgversion) }}
+        config: ["default", "postgis"]
         
     name: Test on PostgreSQL ${{ matrix.pg }}
```

Comment:
Will here better to add `in ${{ matrix.config }} mode`?

Reply:
Thank you for your suggestion. The current description is **name: Test on PostgreSQL ${{ matrix.pg }}, ${{ matrix.config }} mode**.  
I think it's OK for now. Could you check it again and resolve this comment.

Reply:
@MinhLA1410 , I have no "Resolve" button here. Only "Comment". Look like @t-kataym should resolve this change as a person who can approve this PR.

Reply:
Oh. Thank you.